core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.x86_64.txt

```diff
@@ -16,6 +16,7 @@ python-chardet # for createrepo
 python-deltarpm # for createrepo
 python-kitchen # for createrepo
 yum-utils
+python3
```

Comment:
Shouldn't be under `[packages]` section instead of `[packages-repo-prereqs]`?

Reply:
Fixed.